frontend/src/store/teachers.js

```diff
@@ -87,4 +87,49 @@ export const teacherGetters = {
   teachers(state) {
     return state.teachers;
   },
+
+  suggestedTeachers(state) {
+    return (studentId) => {
+
+      // 1. Find the student using the ID
+      const student = state.students.find(student => student.StudentID === studentId)
+      if (!student) {
```

Comment:
```suggestion
      // 1. Find the student using the ID
      const student = state.students.find(student => parseInt(student.StudentID) === partseInt(studentId))
      if (!student) {
```
Suggest wrapping parseInt() around studentId as it was a string